great_expectations/self_check/util.py

```diff
@@ -1125,62 +1169,47 @@ def candidate_test_is_on_temporary_notimplemented_list(context, expectation_type
             "expect_compound_columns_to_be_unique",
             "expect_multicolumn_values_to_be_unique",
             "expect_column_pair_cramers_phi_value_to_be_less_than",
-            # "expect_table_row_count_to_equal_other_table",
             "expect_multicolumn_sum_to_equal",
         ]
+    if context in ["bigquery"]:
+        return expectation_type in [
+            "expect_column_values_to_be_increasing",
+            "expect_column_values_to_be_decreasing",
+            "expect_column_values_to_match_strftime_format",
+            "expect_column_values_to_be_dateutil_parseable",
+            "expect_column_values_to_be_json_parseable",
+            "expect_column_values_to_match_json_schema",
+            "expect_column_stdev_to_be_between",
+            "expect_column_most_common_value_to_be_in_set",
+            "expect_column_bootstrapped_ks_test_p_value_to_be_greater_than",
+            "expect_column_kl_divergence_to_be_less_than",
+            "expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than",
+            "expect_column_chisquare_test_p_value_to_be_greater_than",
+            "expect_column_pair_values_to_be_equal",
+            "expect_column_pair_values_A_to_be_greater_than_B",
+            "expect_column_pair_values_to_be_in_set",
+            "expect_select_column_values_to_be_unique_within_record",
+            "expect_compound_columns_to_be_unique",
+            "expect_multicolumn_values_to_be_unique",
+            "expect_column_pair_cramers_phi_value_to_be_less_than",
+            "expect_multicolumn_sum_to_equal",
+            "expect_column_values_to_be_between",  # unique to bigquery -- https://github.com/great-expectations/great_expectations/issues/3261
+            "expect_column_values_to_be_of_type",  # unique to bigquery -- https://github.com/great-expectations/great_expectations/issues/3261
+            "expect_column_values_to_be_in_set",  # unique to bigquery -- https://github.com/great-expectations/great_expectations/issues/3261
+            "expect_column_values_to_be_in_type_list",  # unique to bigquery -- https://github.com/great-expectations/great_expectations/issues/3261
+            "expect_column_values_to_match_like_pattern_list",  # unique to bigquery -- https://github.com/great-expectations/great_expectations/issues/3261
+            "expect_column_values_to_not_match_like_pattern_list",  # unique to bigquery -- https://github.com/great-expectations/great_expectations/issues/3261
```

Comment:
Are these also noted in Jira/Confluence? If not, would you mind adding it there for improved visibility?

Reply:
GREAT-166 is the one for these